lib/utils/docker.coffee

```diff
@@ -418,3 +428,13 @@ copyQemu = (context) ->
 	.then ->
 		fs.chmod(binPath, '755')
 	.return(binPath)
+
+# if required, change the permissions (mode) of files added to the tar stream
+# This is needed for windows as the execute bit isn't supported there
+chmodTarEntry = (mode) ->
```

Comment:
Can you add a comment linking to the corresponding moby code for this that you've included in the PR description?